test/index.js

```diff
@@ -73,22 +73,25 @@ test('should processor support promise', t => {
     });
 });
 
-test('should support output as path', t => {
-    writeFileSync('fixtures/output-path/output.css', '');
+test('should support output as (non-previously existent)-path', t => {
+    let testpath = 'fixtures/output-path'
+    //This would be the value of 'output:' option
+    let relative_file = 'build/styles/mystyles.css'
+    let fullfile = [testpath,relative_file].join('/')
```

Comment:
Absolutely! That's just meant for the test, to straighforwardly distinguish one path from the other, it could perfectly be just  `fullfile='fixtures/output-path/build/styles/mystyles.css` :)